docs/api/networkinterfaces/CellularInterface.md

```diff
@@ -4,13 +4,13 @@
 
 The CellularBase provides a C++ API for connecting to the internet over a Cellular device.
 
-Arm Mbed OS provides a [reference implementation of CellularBase](http://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_easy_cellular_connection_8h_source.html), which has more information.
+Arm Mbed OS provides a [reference implementation of CellularBase](http://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_cellular_context_8h_source.html), which has more information.
```

Comment:
Query: Why do we link to the same Doxy page three times on the same page? Could we get rid of two?

Reply:
Good point, removed two links.